sirepo/package_data/static/js/genesis.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var srlog = SIREPO.srlog;
+var srdbg = SIREPO.srdbg;
+
+SIREPO.app.config(function() {
```

Comment:
You can add  SIREPO.SINGLE_FRAME_ANIMATION = [...] here so the Parameter and Particle Distribution reports don't have frame control buttons. See opal.js for an example.

Reply:
7754b3feb